scripts/workers/submission_worker.py

```diff
@@ -530,6 +533,14 @@ def run_submission(
     submission.completed_at = timezone.now()
     submission.save()
 
+    if submission.challenge_phase.challenge.is_docker_based:
+        sender_email = settings.CLOUDCV_TEAM_EMAIL
+        user_email = submission.created_by.email
+        template_id = settings.SENDGRID_SETTINGS.get("TEMPLATES").get(
+            "TASK_DONE_NOTIFICATION"
+        )
+        send_email(sender_email, user_email, template_id)
+
```

Comment:
`submission_worker.py` will not be evaluating submissions for docker/code upload based challenge.